config/crd/test/ray.io_rayclusters.yaml

```diff
@@ -0,0 +1,14234 @@
+---
```

Comment:
Can we use a reference to import these CRDs as we do for MCAD: https://github.com/project-codeflare/codeflare-operator/blob/1264faabc1835b3aafb5e11fc31c4b1bbb9a1382/config/crd/mcad/kustomization.yaml#L4C6-L4C83

Reply:
We only need it for testing and I couldn't get envtest to work with remote files. I'm thinking of changing it so that it downloads them and cleans them up as part of the `BeforeSuite` and `AfterSuite`. It really only needs the `route` and the `raycluster`. Do we need to have the Ray CRDs applied as well before starting the controller?

Reply:
Ah I missed that envtest cannot use Kustomize reference mechanism.

For the dependency on the RayCluster API and the KubeRay CRD, I have some ideas that I need to formalise, but in the short term, what we can do is only start the controller when the API is present in the cluster.

Reply:
Right, but on the other hand, the CFO will never work if KubeRay is not installed. I don't think there is an ideal solution to this problem in the short term. We can document the CFO has to be restarted after KubeRay is installed, until we rework the deployment strategy.

My current thinking is that this controller should be deployed along side KubeRay, e.g. in a sidecar container.